lib/gracenote/request/search.rb

```diff
@@ -24,6 +20,16 @@ def search(artist: nil, album: nil, track: nil, mode: nil, range: nil)
           Option.new(xml, options).render
         end
       end
+
+      private
+
+      def set_range(xml, range)
+        range[:end] ||= range[:start] + 20
+        xml.RANGE {
```

Comment:
__[rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)